src/util/isEventEmitter.ts

```diff
@@ -0,0 +1,15 @@
+export type EventEmmitterLike1 = {
```

Comment:
I wouldn't say it's 'premature', but in general I'd like to avoid expose if there isn't any common usage. If it's certain this will be used in other places, then it should be exported though. Do you have expectations this exported type (`EventEmmitterLikeN`, etcs) being used elsewhere than `fromEvent`?